### DISCUSSION TITLE
While using 'stb_image_write. h' Header file i'm getting a Linker issue

### DISCUSSION THREAD

**Discussion by yashpickles** [nothings/stb#1428] [Q&A]
I've been running DPC++ version 2023.0.0 for my work-related purposes and as an image reader and writer stb library was suggested to me. But while running a basic code I'm getting a Linker Error similar to the one I've attached below. Previously. my teammates were also using stb library to load and write their images on DPC++ version 2022 but they weren't getting any issues or errors. Can anyone here help look into this matter? Would be mega if anyone here would help me out!
![error_1](https://user-images.githubusercontent.com/55975678/209931240-1ca2cbdf-3fe0-4022-8c64-83eaa3ef642e.PNG)

**Comment by nothings** [comment]
I don't have any idea what dpc++ is, but googling turns up this page which maybe implies you need to include "mathimf.h" instead of "math.h"? [Other](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-dpcpp-cpp-compiler-dev-guide-and-reference/top/optimization-and-programming/intel-oneapi-dpc-c-compiler-math-library/use-the-intel-oneapi-dpc-c-compiler-math-library.html)
